src/fetch/fetch.js

```diff
@@ -0,0 +1,17 @@
+function _f(url) {
+  return fetch(url)
+    .then(response => response.json())
+    .catch((err) => {
+      console.error(err)
```

Comment:
`console.error`?
Finally this helper is always resolved, sometimes with `undefined`.
Is it the expected behavior?